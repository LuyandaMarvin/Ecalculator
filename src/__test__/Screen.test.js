import React from "react";
import Screen from "../components/Screen"
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CalcProvider from "../context/CalcContext";

configure({ adapter: new Adapter() });

describe('<Screen />', () => {
    const screen = shallow(
        <CalcProvider>
             <Screen />
        </CalcProvider>
      );
  
    it('renders the correct html output', () => {
      expect(screen.html()).toMatchSnapshot();
    });
});