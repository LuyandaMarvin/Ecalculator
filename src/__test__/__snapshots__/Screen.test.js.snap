// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Screen /> renders the correct html output 1`] = `"<div class=\\"screen\\"><div style=\\"display:inline-block;white-space:nowrap\\">0</div></div>"`;
